### PR TITLE
Add vidio.com & zhycorp.com website

### PR DESCRIPTION
**Pull request by KurokuTetsuya**
# Presence **[vidio.com](https://vidio.com)**
- Track user paused the video or no
- Track duration of video or live streaming
- Track user viewing series
- Track user viewing dashboard account
- Track user viewing categories
- Track user searching a video/series
- Track user viewing packages pricing
Proof it works:
image

# Presence **[zhycorp.com](https://zhycorp.com)**
- Track user viewing homepage
- Track user viewing hastebin
- Track user writing in hastebin
- Track user reading in hastebin
- Track user viewing bot listing
- Track user viewing the details of bot
- Track user viewing staff list page
- Track user viewing supporter list page
Proof it works:
image

### CODE REVIEW

**Comment by mzrtamp** [comment]
+1

**Comment by Tiramitzu** [comment]
+1

File: websites/Z/Zhycorp/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    "service": "Zhycorp",
+    "description": {
+      "en": "Zhycorp is a fairly active and growing community, filled with imaginative members, which might make this community even more interesting."
+    },
+    "url": [
+        "zhycorp.com",
+        "bin.zhycorp.com"
+    ],
+    "version": "2.0.0",
```
Comment (Bas950) [inline websites/Z/Zhycorp/dist/metadata.json:14]: v1.0.0 on release

File: websites/Z/Zhycorp/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "logo": "https://i.imgur.com/YR1SNzp.png",
+    "thumbnail": "https://i.imgur.com/CxO5lUg.png",
+    "color": "#ffffff",
+    "tags": [
```
Comment (Bas950) [inline websites/Z/Zhycorp/dist/metadata.json:18]: not sure if you really want to use FFFFFF here... Doesnt really fit with the logo or anything

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by mzrtamp — DISMISSED** [review]

**Comment by ririxidev** [comment]
zhycorp.com is not 2 months old. Closing this pr and open pull request without zhycorp presence :)

**Comment by KurokuTetsuya** [comment]
> zhycorp.com is not 2 months old. Closing this pr and open pull request without zhycorp presence :)
No, actually the domain is zhycorp.xyz and now its moved to zhycorp.com
You can open zhycorp.xyz and it'll redirected to zhycorp.xyz (Also check the registration date too)

**Comment by ririxidev** [comment]
next time maybe say that if you know that sites need to be 2 months old lmao

**Comment by KurokuTetsuya** [comment]
> next time maybe say that, if you know that sites need to be 2 months old lmao

I don't know if the requirements is 2 months old website

**Comment by ririxidev** [comment]
so you didn't even bother to read guidelines
Screenshot 2021-01-04 at 15 21 04

**Comment by KurokuTetsuya** [comment]
> so you didn't even bother to read guidelines
> Screenshot 2021-01-04 at 15 21 04

Lmao, i'm just follow the structures guidelines

**Comment by mzrtamp** [comment]
Bruh, it just a domain migration, site is more than 2 months old, try to check [zhycorp.xyz](https://zhycorp.xyz) (where this domain registration date is more than 2 months) then you will forwarded to [zhycorp.com](https://zhycorp.com) directly with 301 😐

**Comment by ririxidev** [comment]
Don't need to write it 2nd time. He wrote it I checked, there wasn't any info about .xyz domain so I checked only .com domain :)

**Review by Bas950 — APPROVED** [review]

**Comment by mzrtamp** [comment]
> Don't need to write it 2nd time. He wrote it I checked, there wasn't any info about .xyz domain so I checked only .com domain :)

Bcd njg

File: websites/V/Vidio/presence.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
+const presence = new Presence({
+  clientId: "795564487910227968"
+}),
+presenceData: PresenceData = {
+  largeImageKey: "vidio"
+};
+
+presence.on("UpdateData", async () => {
+  presenceData.startTimestamp = Math.floor(Date.now() / 1000);
```
Comment (ririxidev) [inline websites/V/Vidio/presence.ts:9]: same as other pr.

File: websites/Z/Zhycorp/presence.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
+const presence = new Presence({
+  clientId: "795629338401832990"
+}),
+presenceData: PresenceData = {
+  largeImageKey: "zhycorp"
+};
+
+presence.on("UpdateData", async () => {
+  presenceData.startTimestamp = Math.floor(Date.now() / 1000);
```
Comment (ririxidev) [inline websites/Z/Zhycorp/presence.ts:9]: same as other pr.

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
Okay now, I had more time to investigate [this site](https://zhycorp.com). Sorry but unfortunately, we as Reviewer them decided to not accept this site since:

- This site is low quality.
- Site is still under heavy developement - Is not done (Screenshots below)
image
image
- Looks more as joke, not as real site.

Because of this, we please you to remove https://zhycorp.com website from commits, so we could merge it. With it we are not going to merge it.

***TL:DR**: We are not going to take part in any conversation there with words like: "why this site isn't accepted etc.". It's your choice, and we decide what site is High Quality and what site is not. 

Greetings,
PreMiD Reviewers Team.

**Comment by mzrtamp** [comment]
> Okay now, I had more time to investigate [this site](https://zhycorp.com). Sorry but unfortunately, we as Reviewer team decided to not accept this site since:
> 
> * This site is low quality.
> * Site is still under heavy developement - Is not done (Screenshots below)
>   image
>   image
> * Looks more as joke, not as real site.
> 
> Because of this, we please you to remove https://zhycorp.com website from commits, so we could merge other site. With zhycorp presence we are not going to merge it.
> 
> _**TL:DR**: We are not going to take part in any conversation there with words like: "why this site isn't accepted etc.". It's our choice, and we decide what site is High Quality and what site is not._
> 
> Greetings,
> PreMiD Reviewers Team.

***Site is still under heavy developement - Is not done (Screenshots below)***
No, it's not. What you provided is indeed the design and the main purpose of that site

*I'm just commenting, it doesn't need any additional decision.*

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]
